onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -91,25 +92,30 @@
 
     private static final int NORTH = 0;
 
-    private static final int NORTH_WEST = 1;
```

Comment:
IIRC the const direction vars were only used to index into the vehicle_icons array. I redefined the numeric indexes to be consistent with the halfWinds generated. I will double check when I get the chance.